lib/commands/build.js

```diff
@@ -10,9 +9,9 @@ module.exports = Command.extend({
   aliases: ['b'],
 
   availableOptions: [
-    { name: 'environment', type: String,  default: 'development', aliases: ['e', { 'dev': 'development' }, { 'prod': 'production' }] },
-    { name: 'output-path', type: path,    default: 'dist/',       aliases: ['o'] },
-    { name: 'watch',       type: Boolean, default: false,         aliases: ['w'] },
+    { name: 'environment', type: String,             default: 'development', aliases: ['e', { 'dev': 'development' }, { 'prod': 'production' }] },
+    { name: 'output-path', type: function Path() {}, default: 'dist/',       aliases: ['o'] },
```

Comment:
i will push "my version" for now so that you can have a look at it but again, if you object it, i will change it to a pathOptionType.